api/security/api-key-authorize.js

```diff
@@ -18,7 +18,7 @@ module.exports = function authorize (request, response, next) {
         } else {
           requestedResource = request.body['uri']
         }
-        if (isAuthorized(availableScopes, requestedResource)) {
+        if (isSuperAdmin || isAuthorized(availableScopes, requestedResource)) {
```

Comment:
If I am not a super admin, then `isAuthorized` will throw an error when it tries to lower case a null/undefined value (since we are not passing a URI to this endpoint).